data/org.freedesktop.portal.Flatpak.xml

```diff
@@ -228,6 +228,27 @@
       <arg type='b' name='to_process_group' direction='in'/>
     </method>
 
+    <!--
+        SpawnRelativePid:
```

Comment:
This name looks like it spawns something. I think a better name would be something like GetSpawnedPid or something.